### ISSUE TITLE
The 'SDS' part in the test names (A.03.IR05.SDS) d...

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-invocable-sds#8]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The 'SDS' part in the test names (A.03.IR05.SDS) does not apply anymore as the standard ISO19139 schemas are used

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the future work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
